rollingpin/args.py

```diff
@@ -1,13 +1,23 @@
 import argparse
+import os
+import sys
 
 
 PAUSEAFTER_DEFAULT = 1
 
 
 class ExtendList(argparse.Action):
-
     def __call__(self, parser, namespace, values, option_string):
         list_to_extend = getattr(namespace, self.dest)
+        if list_to_extend is self.default:
+            # if the current value is the same ref as the default,
+            # create a copy (so that extensions does not modify the
+            # default value
+            list_to_extend = self.default[:]
```

Comment:
This is a better solve for the problem, some more background is in the commit message, but, ultimately, it may be a bug in the stdlib module itself.  I may spend some time trying to reproduce in a short snippet and put in a bug report, but doubtful that they will do anything about it (the codebase for this is an absolute mess).